### PR TITLE
Add classnames to toolbar buttons

### PR DESCRIPTION
**Pull request by cheryllaird**

### CODE REVIEW

File: dist/esm/modules/Toolbar.js [hunk 1]
```diff
@@ -66,6 +68,7 @@ export class Toolbar extends BaseModule {
 			},
 			{
 				icon: IconAlignRight,
+				class: "ql-image-align-right",
```
Comment (goofballLogic) [inline dist/esm/modules/Toolbar.js:71]: What's the benefit of the `ql-image-` prefix?

File: dist/umd/image-resize.min.js [hunk 2]
```diff
@@ -1,851 +1 @@
...
...asAttribute(o)&&(t[o]=e.getAttribute(o)),t},{})}}]),t}(m);window.Quill.register(b,!0),"function"!=typeof Object.assign&&(Object.assign=function(e){if(null==e)throw new TypeError("Cannot convert undefined or null to object");e=Object(e);for(var t=1;t<arguments.length;t++){var o=arguments[t];if(null!=o)for(var n in o)Object.prototype.hasOwnProperty.call(o,n)&&(e[n]=o[n])}return e}),window.Quill.register("modules/imageResize",y)}},function(e,t){e.exports='<svg viewbox="0 0 18 18">\n  <line class="ql-stroke" x1="15" x2="3" y1="9" y2="9"></line>\n  <line class="ql-stroke" x1="14" x2="4" y1="14" y2="14"></line>\n  <line class="ql-stroke" x1="12" x2="6" y1="4" y2="4"></line>\n</svg>'},function(e,t){e.exports='<svg viewbox="0 0 18 18">\n  <line class="ql-stroke" x1="3" x2="15" y1="9" y2="9"></line>\n  <line class="ql-stroke" x1="3" x2="13" y1="14" y2="14"></line>\n  <line class="ql-stroke" x1="3" x2="9" y1="4" y2="4"></line>\n</svg>'},function(e,t){e.exports='<svg viewbox="0 0 18 18">\n  <line class="ql-stroke" x1="15" x2="3" y1="9" y2="9"></line>\n  <line class="ql-stroke" x1="15" x2="5" y1="14" y2="14"></line>\n  <line class="ql-stroke" x1="15" x2="9" y1="4" y2="4"></line>\n</svg>'}])});
```
Comment (goofballLogic) [inline dist/umd/image-resize.min.js:1]: Looks like this wasn't minified previously (despite the name). Probably ok to do so now?

File: dist/esm/modules/Toolbar.js (hunk 1 above)
Comment (cheryllaird) [reply dist/esm/modules/Toolbar.js:71]: I think `align-right` would be too generic and may clash easily. I followed the convention of quill who set `ql-align-right` and made it `ql-image-` for this package

Comment (goofballLogic) [reply dist/esm/modules/Toolbar.js:71]: is the concern that `align-right` might be targetted by someone else's CSS ?

Comment (goofballLogic) [reply dist/esm/modules/Toolbar.js:71]: I'll approve for now, but generally I advocate for semantic CSS names. I think the problems with styling clashes are often overstated.

**Review by goofballLogic — APPROVED** [review]

**Pull request merged**